ingestors/kafka/src/main/scala/hydra/kafka/programs/CreateTopicProgram.scala

```diff
@@ -155,8 +155,11 @@ final class CreateTopicProgram[F[_]: Bracket[*[_], Throwable]: Sleep: Logger] pr
     val td = createTopicRequest.numPartitions.fold(defaultTopicDetails)(numP =>
       defaultTopicDetails.copy(numPartitions = numP.value))
       .copy(partialConfig = defaultTopicDetails.configs ++ getCleanupPolicyConfig)
+
     (for {
-      _ <- Resource.eval(validator.validate(createTopicRequest, topicName, withRequiredFields))
+      metadata <- Resource.eval(metadataAlgebra.getMetadataFor(topicName))
+      createdDate = metadata.map(_.value.createdDate).orElse(Option(createTopicRequest.createdDate))
+      _ <- Resource.eval(validator.validate(createTopicRequest, topicName, withRequiredFields, createdDate))
```

Comment:
You added changes to validation but I don't see changes in `CreateTopicProgramSpec` tests

Reply:
Added tests to CreateTopicProgramSpec.